dask/array/core.py

```diff
@@ -3187,6 +3224,16 @@ def cov(m, y=None, rowvar=1, bias=0, ddof=None):
         return (dot(X, X.T.conj()) / fact).squeeze()
 
 
+@wraps(np.corrcoef)
+def corrcoef(x, y=None, rowvar=1):
+    c = cov(x, y, rowvar)
+    if c.shape == ():
+        return c/c
+    d = diag(c)
+    d = d.reshape((d.shape[0], 1))
+    return c / sqrt(d * d.T)
```

Comment:
This might work fine in the single-machine scheduler.  The other way with multiple divisions works more robustly I think, and so might be helpful to have in the distributed scheduler, which is in a bit more of a challenging environment.